src/document/json/rga_tree_list.ts

```diff
@@ -245,6 +245,37 @@ export class RGATreeList {
     }
   }
 
+  /**
+   * `moveFront` moves the given element
+   * at the first of array
+   */
+  public moveFront(createdAt: TimeTicket, executedAt: TimeTicket): void {
+    const node = this.nodeMapByCreatedAt.get(createdAt.toIDString());
+    if (!node) {
+      logger.fatal(`cant find the given node: ${createdAt.toIDString()}`);
+    }
+
+    let prevNode = node!.getPrev();
+
+    do {
+      if (prevNode) {
+        prevNode = prevNode.getPrev();
+      }
+    } while (prevNode && prevNode.getPrev());
+
+    if (!prevNode) {
+      logger.fatal(`cant find the given node: ${createdAt.toIDString()}`);
+    }
+
+
+    console.log("preNode", prevNode);
+    console.log("targetNode", node);
```

Comment:
RGATreeList represents the position of the first node in the linked list as `dummyHead` with no value. It would be nice to set `prevNode` to `dummyHead` right away.

```suggestion
    const prevNode = this.dummyHead;
```